tools/theme/README.md

```diff
@@ -54,9 +51,49 @@ import '@spectrum-web-components/theme/express/scale-medium.js';
 import '@spectrum-web-components/theme/express/scale-large.js';
 ```
 
+When looking to leverage the `Theme` base class as a type and/or for extension purposes, do so via:
+
+```
+import { Theme } from '@spectrum-web-components/theme';
+```
+
+## What it's doing
+
+Visually, all Spectrum Web Components are an expression of the design tokens specified by Spectrum, Adobe's design system. On the web, these tokens are applied as CSS Custom Properties. Using `sp-theme` as a parent element for your components ensures that those elements' styles are not only consistent with the color and scale you've specified in your particular instance, but for _all_ the other color, scale, and content direction specifications across Spectrum.
```

Comment:
```suggestion
Visually, all Spectrum Web Components are an expression of the design tokens specified by Spectrum, Adobe's design system. On the web, these tokens are made available as CSS Custom Properties. Using `sp-theme` as a parent element for your components ensures that those CSS Custom Properties can be leveraged by the elements within your application. This ensure a consistent delivery of the color and scale you've specified in your particular instance, as well as for _all_ the other color, scale, and content direction specifications across Spectrum.
```

Words are hard.

Reply:
> This ensure a consistent delivery of the color and scale you've specified in your particular instance, as well as for _all_ the other color, scale, and content direction specifications across Spectrum.

I think the wording of this sentence is still a little strange, so I kept what I had originally. 

> This ensures consistent delivery of not only the color and scale you've specified in your particular instance, but for _all_ the other color, scale, and content direction specifications across Spectrum.